.github/workflows/ci.yml

```diff
@@ -0,0 +1,104 @@
+name: infoware CI
+on:
+  push:
+  pull_request:
+
+jobs:
+  linux-x64:
+    name: Linux x64 (${{matrix.cfg.cpp-version}})
+    runs-on: ${{matrix.cfg.os}}
+    strategy:
+      fail-fast: false
+      matrix:
+        cfg:
+          - { os: ubuntu-18.04, cpp-version: g++-8 }
+          - { os: ubuntu-18.04, cpp-version: g++-9 }
+          - { os: ubuntu-18.04, cpp-version: clang++-7 }
+          - { os: ubuntu-18.04, cpp-version: clang++-8 }
+          - { os: ubuntu-18.04, cpp-version: clang++-9 }
```

Comment:
I don't think there's much point in doing this. 18.04 is the current(?) LTS, the compilers are already ancient (esp. Clang!) – I'd just go for the default virtuals (AFAICT, this can be achieved by Just setting this to `g++`, `clang++`?)

Reply:
Sure. My other project preferred putting os in a variable, since if we want to update a single os and continue testing, it won't affect the other. That project caters to devs who may be on older software/hardware, so that's where that came from.

Switched to use only the default preinstalled g++ and clang++ compilers, and `ubuntu-18.04` on all workers.